src/components/SigningModal/TransactionJourney.js

```diff
@@ -159,7 +170,8 @@ function ProgressLine({ status, pulse, themeColor }) {
           border-left: 1px dashed ${theme.surfaceOpened};
         }
 
-        ${status !== NEXT && highlight}
+        ${status === PREVIOUS && highlight}
+        ${status === CURRENT && pulse && highlight}
```

Comment:
I wonder if we could simplify the above, so it's a bit more clear when the `highlight` is being applied:

```js
${if (status === PREVIOUS || status === CURRENT && pulse) {
  return highlight
}}
```